spyder/plugins/application/plugin.py

```diff
@@ -97,7 +100,10 @@ def on_editor_available(self):
         editor = self.get_plugin(Plugins.Editor)
         self.get_container().sig_load_log_file.connect(editor.load)
 
-    def on_close(self):
+    # -------------------------- PLUGIN TEARDOWN ------------------------------
+    # @on_plugin_teardown(plugin=Plugins.)
+
+    def on_close(self, _unused=True):
```

Comment:
Why do we need to set this parameter `_unused` to `True`? Can it simply be `_unused`?

Reply:
Both work, actually, that `_unused=True` is a leftover of the previous `cancellable` parameter.

Reply:
Ok, then I think we can leave it.